src/importexport/musicxml/internal/musicxml/import/importmusicxml.h

```diff
@@ -22,11 +22,13 @@
 
 #pragma once
 
+#include "dom/score.h"
 #include "global/types/string.h"
 #include "engravingerrors.h"
 
-namespace mu::engraving {
-class Score;
-
+using namespace mu::engraving;
+namespace mu::iex::musicxml {
 Err importMusicXmlfromBuffer(Score* score, const muse::String&, const muse::ByteArray& data);
+Err importCompressedMusicXml(MasterScore* score, const String& name, bool forceMode);
+Err importMusicXml(MasterScore* score, const String& name, bool forceMode);
```

Comment:
The slightly more correct way would be:
```c++
namespace mu::engraving {
class Score;
}

namespace mu::iex::musicxml {
Err importMusicXmlfromBuffer(engraving::Score* score, const muse::String&, const muse::ByteArray& data);	Err importMusicXmlfromBuffer(engraving::Score* score, const muse::String&, const muse::ByteArray& data);
Err importCompressedMusicXml(engraving::MasterScore* score, const String& name, bool forceMode);
Err importMusicXml(engraving::MasterScore* score, const String& name, bool forceMode);
}
```

Officially, `using namespace` in header files is not really preferable; but in this case it's generally fine because these headers are very internal in the musicxml module and not exposed outside it, plus that it would be _a lot_ of work to avoid the `using namespace mu::engraving`. So inside the `musicxml/internal/musicxml` folder it's fine, but it would be good to keep it in mind.